### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by aljones15**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+Bedrock Non-Commercial License v1.0
+===================================
+
+Copyright (c) 2011-2019 Digital Bazaar, Inc.
```
Comment (gannan08) [inline LICENSE.md]:
```suggestion
Copyright (c) 2011-2022 Digital Bazaar, Inc.
```

**Review by gannan08 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "homepage": "https://github.com/digitalbazaar/bedrock-multikey-context#readme",
+  "dependencies": {
+    "@digitalbazaar/multikey-context": "^1.0.0"
+  },
+  "peerDependencies": {
+    "bedrock": "^4.0.0",
+    "bedrock-jsonld-document-loader": "^1.0.0"
+  },
+  "engines": {
+    "node": ">=12.0.0"
```
Comment (gannan08) [inline package.json]:
```suggestion
    "node": ">=16.0.0"
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#bedrock-multikey-context CHANGELOG
```
Comment (dlongley) [inline CHANGELOG.md]:
```suggestion
# bedrock-multikey-context CHANGELOG
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+#bedrock multikey context
```
Comment (dlongley) [inline README.md]:
```suggestion
# bedrock multikey context
```

File: package.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-multikey-context/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-multikey-context#readme",
+  "dependencies": {
+    "@digitalbazaar/multikey-context": "^1.0.0"
+  },
+  "peerDependencies": {
+    "bedrock": "^4.0.0",
+    "bedrock-jsonld-document-loader": "^1.0.0"
```
Comment (dlongley) [inline package.json]: Needs to be fixed.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: package.json [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  "license": "SEE LICENSE IN LICENSE.md",
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-multikey-context/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-multikey-context#readme",
+  "dependencies": {
+    "@digitalbazaar/multikey-context": "^1.0.0"
+  },
+  "peerDependencies": {
+    "@bedrock/core": "^6.0.0",
```
Comment (aljones15) [inline package.json]: @dlongley @gannan08 should I drop this peer dep of `@bedrock/core` or should I keep it?

Comment (aljones15) [reply package.json]: Removed `@bedrock/core` peerDep here: https://github.com/digitalbazaar/bedrock-multikey-context/pull/1/commits/885fc2561defe9267b9586df7fb989080b8acd6a

File: package.json (hunk 5 above)
Comment (aljones15) [reply package.json]: Addressed the namespace issue here: https://github.com/digitalbazaar/bedrock-multikey-context/pull/1/commits/82b6db4b6524bcd9de00a27b87429afeeed3610a

File: package.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@bedrock/multikey-context",
+  "version": "1.0.0",
```
Comment (dlongley) [inline package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: package.json [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+  "type": "module",
+  "description": "Bedrock loader for multi key context",
+  "main": "./lib/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/bedrock-multikey-context.git"
```
Comment (dlongley) [inline package.json:12]:
```suggestion
    "url": "https://github.com/digitalbazaar/bedrock-multikey-context.git"
```

**Review by dlongley — APPROVED** [review]
Approving -- and I'll fix the package.json issues, just noting them here for you to see.

**Pull request merged**